src/cljs/main/broadfcui/common.cljs

```diff
@@ -1,10 +1,10 @@
-(ns org.broadinstitute.firecloud-ui.common
+(ns broadfcui.common
```

Comment:
Thank you. I believe I have warned everyone who cares except for David An, who is a reviewer on this PR. I've also squashed the commits to four and I'm going to merge them without squashing so the steps are clearer and the rename is on its own.